srv/cat-service.cds

```diff
@@ -1,10 +1,41 @@
-using { my.bookshop as my } from '../db/schema';
+using {my.bookshop as my} from '../db/index';
 
-@path:'browse'
+@path : 'browse'
 service CatalogService {
+    @readonly
+    entity Books       as projection on my.Books excluding {
+        createdBy,
+        modifiedBy
+    } actions {
+        action addReview(rating : Integer, title : String, text : String) returns Reviews;
+    };
 
-  @readonly entity Books as SELECT from my.Books {*,
-    author.name as author
-  } excluding { createdBy, modifiedBy };
+    @readonly
+    entity Authors     as projection on my.Authors;
 
-}
\ No newline at end of file
+    @readonly
+    entity Reviews     as projection on my.Reviews;
+
+    @readonly
+    entity ListOfBooks as
+        select from Books
+        excluding {
+            descr
+        };
+
+    action submitOrder(book : Books : ID, amount : Integer) returns {
+        stock : Integer
+    };
+
+    // access control restrictions
+    annotate CatalogService.Reviews with @restrict : [
```

Comment:
Is the qualifier `CatalogService` required here, as you define this inside the `CatalogService`?